File: Movil/ProyectoABCKotlin/app/src/main/java/com/example/proyectoabckotlin/pojo/Usuario.kt
package com.example.proyectoabckotlin.pojo

class Usuario {
    var usuario: String? = null
    var contrasena: String? = null
    var token: String? = null
    var tipoUsuario: String? = null

    fun crearUsuario(usuario:String, contrasena:String): Usuario {
        val nuevoUsuario = Usuario()
        nuevoUsuario.usuario = usuario
        nuevoUsuario.contrasena = contrasena
        return nuevoUsuario
    }
}

File: Web/src/app/candidato/usuario.ts
export class Usuario {
    id: number;
    usuario: String;
    contrasena: String;
    tipoUsuario: String;
  
      constructor(
        id: number,
        usuario: String,
        contrasena: String,
        tipoUsuario: String,
      ) {
          this.id = id;
          this.usuario = usuario;
          this.contrasena = contrasena;
          this.tipoUsuario = tipoUsuario;

      }
  }